pkg/controller/controller.go

```diff
@@ -324,11 +345,37 @@ func sendData(data *bytes.Buffer, contr Controller){
 	}
 	//servers.mux.Unlock()
 }
-func StartController(configFile string, servers []string, events [] string){
-     api, err := CreateK8sApiserverClient(configFile) 
+func validateKubeClusterFields(configFile, kubeURL, kubeServAcctToken string) bool {
+	if configFile == "" && kubeURL == "" && kubeServAcctToken == "" {
+		fmt.Println("Using inbuilt POD values field for Kube client configuration")
+		return true
+	}
+	if configFile == "" && kubeURL != "" && kubeServAcctToken != "" {
+		fmt.Println("Using kubeURL and kubeServAcctToken field for Kube Client configuration")
+		return true
+	}
+	if configFile != "" {
+		fmt.Println("Using configFile for configuration")
+		return true
+	}
+	if configFile == "" && kubeURL != "" && kubeServAcctToken == "" {
+		fmt.Println("Kubernetes Cluster Service Account Token Not Present: Connection Parameters invalid")
```

Comment:
Are we handling any http status code for this ?

Reply:
Done